src/Clock.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+export class Clock extends React.Component {
+  state = {
+    time: new Date().toTimeString().replace(/ .*/, ''),
```

Comment:
```suggestion
    time: new Date().toLocaleTimeString(),
```
Можна використати просто метод toLocaleTimeString